### ISSUE TITLE
Repoint Run 1.1, 1.2 base_dirs to `/global/projecta/projectdirs/lsst/production/`

### ISSUE THREAD

**Issue by wmwv** [LSSTDESC/gcr-catalogs#301]
As part of standardizing directory names the data products are now in:
`/global/projecta/projectdirs/lsst/production/DC2_PhoSim/Run1.1p`
`/global/projecta/projectdirs/lsst/production/DC2_PhoSim/Run1.2p`
`/global/projecta/projectdirs/lsst/production/DC2_ImSim/Run1.2i`

Copy the data products if necessary to make sure the latest ones are in the correct directory.

**Comment by wmwv** [comment]
I've inserted a `dpdd` directory to contain the catalogs.

**Comment by wmwv** [comment]
This is done, but blocked on #302.

**Comment by heather999** [comment]
Now that #302 is closed, is it possible to finish this issue off too?

**Comment by yymao** [comment]
@wmwv have created a branch for this but he's traveling now. @heather999 how urgent is this? Should we pick up @wmwv's work or just wait a few more days?

**Comment by heather999** [comment]
A few days is fine - I just need to clean up `projecta` to make more room for Run2.1i data.  Completion of this issue will allow me to remove duplicates of some of the DM processing data.

**Comment by wmwv** [comment]
I've just now fully closed out #302.  The code was fine, but I needed to create the schema files.

**Comment by wmwv** [comment]
PR submitted.  #306 

We should check the tutorials in `DC2-analysis` to see if there are any exposed paths.

**Comment by wmwv** [comment]
Yes, there are some.  Issue created on `DC2-analysis`
https://github.com/LSSTDESC/DC2-analysis/issues/83
